pyproject.toml

```
@@ -0,0 +1,87 @@
+[build-system]
+requires = ["setuptools", "pip >= 10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pysatMissions"
+version = "0.3.4"
+description = "Mission Planning toolkit for pysat"
+readme = "README.md"
+requires-python = ">=3.8"
```

This may need to be updated
```suggestion
requires-python = ">=3.6"
```